### PR TITLE
realtime updates to QUEST_LV_0200 1/2

### PR DESCRIPTION
**Pull request by terrapin47**
Realtime updates to the questline of pilgrim's path ( all areas East of Fedimian ).

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -2800,16 +2800,16 @@ QUEST_LV_0200_20150317_002799	{memo X}There are altars with unknown purposes in
...
+QUEST_LV_0200_20150317_002803	Defeat Merge
 QUEST_LV_0200_20150317_002804	Cool the altar
-QUEST_LV_0200_20150317_002805	The altar in Oshimas Route keeps fuming red light. Go and check it.
-QUEST_LV_0200_20150317_002806	Spray the cold fluid of monster
+QUEST_LV_0200_20150317_002805	The altar at Oshimas Highway keeps producing a red light. Go and check on it.
+QUEST_LV_0200_20150317_002806	Apply the cold monster fluid.
 QUEST_LV_0200_20150317_002807	{memo X}You can use the cold fluid rarely found in monsters to cool down the wrath of the altar in Oshimas Route. Defeat the monsters in Scattered Stairway and get the cold fluid.
 QUEST_LV_0200_20150317_002808	Spray the monster fluid
-QUEST_LV_0200_20150317_002809	Spray the fluid on Oshimas Altar to cool it down
+QUEST_LV_0200_20150317_002809	Spray the fluid on the Oshimas Highway Altar to cool it down
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2809]: Spray the fluid on the Oshimas Highway Altar to cool it down -> Spray the fluid on the altar at Oshimas Highway to cool it down

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -3210,113 +3210,113 @@ QUEST_LV_0200_20150323_003209	{memo X}You should not touch it.{nl}It will come s
 QUEST_LV_0200_20150323_003210	Did you see it? That boy is my son who left for the pilgrimage 4 years ago.{nl}I followed him to save him from the curse.
 QUEST_LV_0200_20150323_003211	{memo X}Thank you so much for your words.{nl}It will be good if you could defeat the monsters that are disturbing me.
 QUEST_LV_0200_20150323_003212	There is a way to save my son.{nl}I believe myself that my truth will be undertood someday.
-QUEST_LV_0200_20150323_003213	Well done. {nl}I am not scared of anything from now on because of you.
-QUEST_LV_0200_20150323_003214	If the amount of the leftover food that I gave to you is not enough, go to Ishiblashi square.{nl}I have mixed divine water into the food there.
+QUEST_LV_0200_20150323_003213	Well done. {nl}I won't be scared of anything from now on, because of you.
+QUEST_LV_0200_20150323_003214	If the amount of the leftover food that I gave to you is not enough, go to Ishiblysh square.{nl}I have mixed divine water into the food there.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3214]: Ishiblysh square -> Ishiblysh Square

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -3210,113 +3210,113 @@ QUEST_LV_0200_20150323_003209	{memo X}You should not touch it.{nl}It will come s
...
-QUEST_LV_0200_20150323_003216	To be honest with you, I don't know if I would be able to save my son with this.{nl}Hope I mean.. I need some reliable result.
+QUEST_LV_0200_20150323_003216	To be honest with you, I don't know if I will be able to save my son with this.{nl}Hope, I mean.. I need some reliable result.
 QUEST_LV_0200_20150323_003217	When you throw this leftover food to those monsters, there will be some of them who will eat the food.{nl}If you attack them a lot, they will throw up. Bring me that.{nl}
-QUEST_LV_0200_20150323_003218	If there is not enough leftover food, we may get some more from Ishiblashi square.
-QUEST_LV_0200_20150323_003219	The lump of the curse. It is indeed effective. {nl}Now I should wait for my son to throw up.{nl}Thank you so much. May the blessing of the goddesses be with you always..{nl}{nl}
-QUEST_LV_0200_20150323_003220	I am sorry, but I am a bit busy.{nl}I have so many things going in my head so could you just please go now?
-QUEST_LV_0200_20150323_003221	What the hell..{nl}You saved me. I am sorry for while ago.
+QUEST_LV_0200_20150323_003218	If there is not enough leftover food, we may get some more from Ishiblysh square.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3218]: Ishiblysh square -> Ishiblysh Square

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -3210,113 +3210,113 @@ QUEST_LV_0200_20150323_003209	{memo X}You should not touch it.{nl}It will come s
...
+QUEST_LV_0200_20150323_003219	The condensed curse energy. It is indeed effective. {nl}Now I should wait for my son to throw up.{nl}Thank you so much. May the blessing of the goddesses be with you always..{nl}{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3219]: blessing of the goddesses -> blessing of the Goddesses?

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -3210,113 +3210,113 @@ QUEST_LV_0200_20150323_003209	{memo X}You should not touch it.{nl}It will come s
...
+QUEST_LV_0200_20150323_003270	May the blessing of the Goddesses be with you always.{nl}-- Sanctum Manager : [Druid Master] Jina Green{nl}
 QUEST_LV_0200_20150323_003271	If you prove your belief by making contributions, you will receive the blessing.{nl}For the advices on the belief, please purchase warp scrolls from the stand and visit.{nl}
-QUEST_LV_0200_20150323_003272	Lower yourself in front of the goddesses and leave everything to her.{nl}The goddesses will bless the ones who truly confess their wrongdoings.{nl}
-QUEST_LV_0200_20150323_003273	May the goddesses remove your ordeals.{nl}-- Sanctum Manager : [Krivi Master] Hercus
-QUEST_LV_0200_20150323_003274	Make contributions to appreciate the goddesses.{nl}The blessing will be with you always.
-QUEST_LV_0200_20150323_003275	Pray in front of the goddesses.{nl}You should dedicate the vicious beings for sacrifices.
+QUEST_LV_0200_20150323_003272	Lower yourself in front of the Goddesses and leave everything to them.{nl}The Goddesses will bless the ones who truly confess their wrongdoings.{nl}
+QUEST_LV_0200_20150323_003273	May the Goddesses remove your ordeals.{nl}-- Sanctum Manager : [Krivi Master] Hercus
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3273]: Krivi Master -> Krivis Master

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**